core-web/libs/sdk/client/src/lib/editor/models/listeners.model.ts

```diff
@@ -4,32 +4,31 @@
  * @export
  * @enum {number}
  */
-export declare enum NOTIFY_CUSTOMER {
+export enum NOTIFY_CUSTOMER {
     /**
      * Request to page to reload
      */
-    EMA_RELOAD_PAGE = "ema-reload-page",
+    EMA_RELOAD_PAGE = 'ema-reload-page',
     /**
      * Request the bounds for the elements
      */
-    EMA_REQUEST_BOUNDS = "ema-request-bounds",
+    EMA_REQUEST_BOUNDS = 'ema-request-bounds',
     /**
      * Received pong from the editor
      */
-    EMA_EDITOR_PONG = "ema-editor-pong"
-}
-export interface DotCMSPageEditorConfig {
-    onReload: () => void;
-    pathname?: string;
+    EMA_EDITOR_PONG = 'ema-editor-pong'
 }
+
 interface DotCMSPageEditorListener {
     type: 'listener';
     event: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     callback: (ev: any) => void;
```

Comment:
can we avoid this?

Reply:
We can, but that means creating interfaces for each Listener Event that we are going to use.
I think that for this case, an any is easier to maintain

Reply:
is easier to maintain but is harder to keep track of what are we doing in that callback. We can either make it `unknown` and handle the types where we are using it or create a type with union of types

`type UnionOfTypes = FirstType | SecondType | ThirdType`

This way we will always know what is that callback

Reply:
In my opinion if we leave this as any, it can get out of our hands pretty quickly

Reply:
Done!